content/docs/02-guides/04-o1.mdx

```diff
@@ -155,8 +155,9 @@ The useChat hook on your root page (`app/page.tsx`) will make a request to your
 Ready to get started? Here's how you can dive in:
 
 1. Explore the documentation at [sdk.vercel.ai/docs](/docs) to understand the full capabilities of the AI SDK.
-2. Check out practical examples at [sdk.vercel.ai/examples](/examples) to see the SDK in action and get inspired for your own projects.
-3. Dive deeper with advanced guides on topics like Retrieval-Augmented Generation (RAG) and multi-modal chat at [sdk.vercel.ai/docs/guides](/docs/guides).
-4. Check out ready-to-deploy AI templates at [vercel.com/templates?type=ai](https://vercel.com/templates?type=ai).
+1. Check out our support for the o1 series of reasoning models in the [OpenAI Provider](/providers/ai-sdk-providers/openai#reasoning-models).
+1. Check out practical examples at [sdk.vercel.ai/examples](/examples) to see the SDK in action and get inspired for your own projects.
```

Comment:
typo: it should've been 1. 2. 3. ?

Reply:
no, markdown does automatic counting so 1. 1. 1. is actually preferred (less changes needed)